jobs/schedulers_test.go

```diff
@@ -143,3 +144,11 @@ func TestScheduler(t *testing.T) {
 		wg.Wait()
 	})
 }
+
+func TestRandomDelay(t *testing.T) {
+	cases := []int64{5,10,100}
+	for _, c := range cases {
+		out := getRandomDelay(c)
+		require.Less(t, out.Milliseconds(), c)
```

Comment:
This test will pass even if `getRandomDelay` returns 0 every time. But on the other hand, this check is the only way to avoid a flaky test. So I think it's okay, but I wanted to bring it up just in case.

Reply:
Yep of course :)